### ISSUE TITLE
Brand request: icon-wellfound

### ISSUE THREAD

**Issue by eightball298** [FortAwesome/Font-Awesome#20340]
### Description

Wellfound is a discrete brand & company. I think the request for a icon-brand-wellfound makes sense to add as a new logo for the users of font-awesome that want to reference the talent product that Wellfound runs.

### Official website

https://wellfound.com

### Brand guidelines

https://wellfound.com/logo

### Icon request checklist

- [X] The title starts with "Brand request: ", is followed by the requested brand name
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate
- [X] I have [understood how requests work](https://fontawesome.com/community/leaderboard/brands#faqs)

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community and for this heads-up.

So https://fontawesome.com/icons/angellist?f=brands&s=solid is not valid anymore and should be replaced by `wellfound` with a new logo?

Ref: https://en.wikipedia.org/wiki/AngelList

> AngelList's Syndicate facilitates startup financing with [accredited investors](https://en.wikipedia.org/wiki/Accredited_investor). AngelList Jobs, later rebranded as Wellfound, connect talents with startups, with over 35,000 recruiting companies, more than 2,000,000 candidates and 5 million registered users.

**Comment by eightball298** [comment]
Yep; seems like the same kind of site/company, but it was spun off as a new division after some investment... It'd be great if I could just use a FontAwesome icon instead of the graphic in a <ul> list that already consists of only the former. Thanks!

**Comment by tagliala** [comment]
But https://www.angellist.com/ is still reachable and shows the old logo, can you please share more information?

**Comment by eightball298** [comment]
I'm just an end-user, not affiliated with the company in any way, but this article on their website, ["AngelList Talent is now Wellfound"](https://wellfound.com/blog/angellist-talent-is-now-wellfound#how-it-started), offers a pretty detailed explanation. 

Here's an excerpt - looks like they just split off into two companies that complement one another:

_We started with AngelList Venture, to make it easier for startups to raise money. Our first product connected founders with a list of angel investors–hence the name. Two years later, we launched AngelList Talent to support startups in the next step of their journey–hiring their teams._

**A Few Other Articles:**
- [AngelList Spins Out Jobs Community, Rebrands as Wellfound](https://www.crowdfundinsider.com/2022/11/198763-angellist-spins-out-jobs-community-rebrands-as-wellfound/)
- [Reintroducing: AngelList](https://www.angellist.com/blog/reintroducing-angellist)
- [AngelList Talent is now Wellfound](https://wellfound.com/blog/angellist-talent-is-now-wellfound)

**Comment by Sumukh** [comment]
Chiming in from the AngelList team. AngelList is focused on building infrastructure for investors & startups. It is currently still using the logo. I think it make sense to keep the angellist logo in font-awesome as is for now.

Wellfound is a discrete brand & company. I think the request for a `icon-brand-wellfound` makes sense to add as a new logo for the users of font-awesome that want to reference the talent product that Wellfound runs.

**Comment by tagliala** [comment]
thanks for the clarification

**Comment by sensibleworld** [comment]
So... our backlog of brand requests has reached a point where even we can't pretend we're keeping up. Some of these issues were opened before I had gray hair. And I've had gray hair for a LONG time. On top of that, treating requests as issues has always made it way too hard to see the forest for the trees.

To fix this, we are now managing all brand requests in **GitHub Discussions**. No more +1s or thumbs-ups needed. There's an upvote system built-in.

We are closing existing brand requests as part of this transition, but **we are not saying no** to your request.

**If this brand request is still relevant for you, please re-open it in Discussions** (and feel free to include a link back to this issue):
-> https://github.com/FortAwesome/Font-Awesome/discussions/new?category=brand-requests

And listen. We know this is annoying. If you need to vent, here are some pre-written responses you can use when opening the new discussion:

* Cool cool cool. No worry at all. I love re-requesting a brand that's redesigned three times while this issue was open.
* If an issue closes after seven years and nobody responds, did it ever really exist?
* I see you have chosen violence today. Re-opening per instructions.
* What's next? I have to file a TPS report, too?
* I am filled with the fury of a thousand suns. Why do I have to re-open the discussion? Couldn't you have just bulk converted all of the issues to discussions? What? You couldn't? What do you mean Github removed that feature earlier this year? You'd have to manually click the "convert to discussion" button on thousands of issues, while also ensuring it gets added to the correct category? That's a bunch of bull-phooey.
* No worries.
* This brand has gone the way of the dodo.
